src/components/home-container/index.jsx

```diff
@@ -5,42 +5,51 @@ import Button from "components/base/button";
 import CreateTask from "components/task/create-new";
 import TaskCard from "components/task/existing-task/task-card";
 import FilterBtnContainer from "components/filter-btn-container";
+import NoTaskFound from "components/not-found/task";
 
 function HomeContainer() {
   const tasks = useSelector((state) => state.todo.tasks);
   const [isNewTaskRequested, setIsNewTaskRequested] = useState(false);
 
-  const handleSuccessfulTaskEntry = () => setIsNewTaskRequested(false);
-  const handleCreateBtnClick = () => setIsNewTaskRequested(true);
-
   return (
     <div className={`container mx-auto ${styles.homeWrapper}`}>
       <h1>Add Tasks</h1>
       <div className={`flex justify-between ${styles.buttonContainer}`}>
-        <Button onButtonClick={handleCreateBtnClick}>Create</Button>
+        <Button onButtonClick={() => setIsNewTaskRequested(true)}>
+          Create
+        </Button>
 
         <FilterBtnContainer />
       </div>
 
       <div className="grid grid-cols-3 gap-34px">
```

Comment:
`gap-34px` doesn't make any sense as there is no design system in place. Please use an appropriate class name here and place the gap property there. Also using rem as unit rather than px is highly suggested.